application/config/constants.php

```diff
@@ -99,8 +99,18 @@
 /**
  * Stripe public and private keys for development.
  */
-define('STRIPE_PUBLIC_KEY', 'pk_test_A4wjqVPPn530rgAXv6sHKgSl00opCMVX9A');
-define('STRIPE_PRIVATE_KEY', 'sk_test_OMC00A11yJUakUU4kx6KoGTp0028EYnLBa');
+if (!isset($STRIPE_PUBLIC_KEY)) {
+    define('STRIPE_PUBLIC_KEY', 'pk_test_A4wjqVPPn530rgAXv6sHKgSl00opCMVX9A');
+    define('STRIPE_PRIVATE_KEY', 'sk_test_OMC00A11yJUakUU4kx6KoGTp0028EYnLBa');
+}
+
+if (!isset($MAIL_AUTH_EMAIL)) {
+    $json = file_get_contents("private_keys/auth_props.json");
+    $authProps = json_decode($json, true);
+
+    define('MAIL_AUTH_EMAIL', $authProps["mail.email"]);
+    define('MAIL_AUTH_PASSWORD', $authProps["mail.password"]);
+}
```

Comment:
If this could not be decoded (file not found or file wrong format), an error will be thrown which should be the correct behaviour.